src/roadmap/releases.md

```diff
@@ -99,6 +95,20 @@ deisrel changelog individual $COMPONENT $NEW_RELEASE | pbcopy
 open https://github.com/deis/$COMPONENT/releases/new?tag=$NEW_RELEASE
 ```
 
+### Step 4: Verify the Component is Available
```

Comment:
This step really takes the troubleshooting advice that @vdice provided and moves it from an intro paragraph into the new Step 4. It also puts the burden on the component releaser to double-check that everything succeeded, which will save headaches when Workflow is released. e2e will report on the failure, but as we've seen before, this isn't always noticed until much later.